t/t-credentials.sh

```diff
@@ -290,6 +290,40 @@ begin_test "credentials skips wwwauth[] with option"
 )
 end_test
 
+begin_test "credentials can authenticate with Bearer auth"
+(
+  set -e
+  git credential capability </dev/null | grep "capability authtype" || exit 0
+
+  reponame="auth-bearer-token"
+  setup_remote_repo "$reponame"
+
+  printf "Bearer::token" > "$CREDSDIR/127.0.0.1--$reponame"
+
+  clone_repo "$reponame" "$reponame"
+  git checkout -b new-branch
+
+  git lfs track "*.dat" 2>&1 | tee track.log
+  grep "Tracking \"\*.dat\"" track.log
+
+  contents="b"
+  contents_oid=$(calc_oid "$contents")
```

Comment:
I'm not sure this `contents_oid` variable is used, so perhaps it could be removed?